tasks/main.yml

```diff
@@ -7,7 +7,40 @@
     name: "{{ item }}"
   with_items:
     - epel-release
+
+- name: Ensure Gluster 3.x Centos repository is absent
+  package:
+    name: "{{ item }}"
+    state: absent
+  become: yes
+  with_items:
     - centos-release-gluster
+
+- name: Gluster-4.1 repo
+  yum_repository:
+    name: centos-gluster-4.1
+    description: CentOS Storage SIG repo for gluster 4.1
+    baseurl: http://mirror.centos.org/centos/$releasever/storage/$basearch/gluster-4.1
+    gpgkey: https://raw.githubusercontent.com/CentOS-Storage-SIG/centos-release-storage-common/master/RPM-GPG-KEY-CentOS-SIG-Storage
+    enabled: yes
+  become: yes
+
+- name: Ceph Luminous
+  yum_repository:
+    name: centos-ceph-luminous
+    description: CentOS Storage SIG repo for ceph luminous
+    baseurl: http://mirror.centos.org/centos/7/storage/x86_64/ceph-luminous/
+    gpgkey: https://raw.githubusercontent.com/CentOS-Storage-SIG/centos-release-storage-common/master/RPM-GPG-KEY-CentOS-SIG-Storage
+    enabled: yes
+  become: yes
+
+- name: Install glusterfs-server
+  yum:
+    name: glusterfs-server
+  become: yes
+
+- name: Install glusterfs packages for mount and unmount
+  yum: name=glusterfs-client,glusterfs-rdma,glusterfs-fuse
```

Comment:
nit: Can these packages be added to the package install above for ''Install glusterfs-server''?
nit: Please use yaml syntax rather than all on one line style.

Reply:
Done